src/ralph/security/migrations/0007_auto_20170331_0934.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from datetime import datetime
+
+from django.db import migrations, models
+from django.db.models import Count
+
+
+
+def update_is_patched(apps, schema_editor):
+    SecurityScan = apps.get_model("security", "SecurityScan")
+    is_patched_ids = SecurityScan.vulnerabilities.through.objects.filter(
+        vulnerability__patch_deadline__gt=datetime.now()
+    ).values_list(
+        'securityscan_id', flat=True
+    ).distinct()
+
+    patched_scans = SecurityScan.objects.filter(id__in=is_patched_ids).update(
```

Comment:
in this case, when my sec scan has two vulnerabilities - one with deadline before today, and one with deadline after today, it'll be marked as is_patched, isn't it?